Future.md

```diff
@@ -1,5 +1,7 @@
 # The Future of SARIF
 
+The purpose of this document is to list potential new areas for standardization. These are not guarantees. They are areas we are strongly considering for a future edition of the SARIF standard.
```

Comment:
Suggestion: "These areas are not guarantees."

Reply:
This is part of what is there.  Are you saying replace the whole paragraph and say only this?

Reply:
No, I'm saying to add "areas" after using the word "These" so that the reader knows what "These" refers to:
The purpose of this document is to list potential new areas for standardization. These areas are not guarantees. They are areas we are strongly considering for a future edition of the SARIF standard.

Reply:
Ah, thanks.  Yes, that's a good suggestion.  I read right past the word "areas" and didn't see it.

Reply:
Then I'd be using the word "areas" three times.  I'll think of something similar.

Reply:
After thinking about it, the paragraph as a whole seems to work better the way it is, which avoids excessive repetition.  If we had that sentence in isolation, then the change would work better.